infrastructure/modules/auth/db.tf

```diff
@@ -146,6 +146,22 @@ resource "aws_dynamodb_table_item" "test_client_user_admin_permissions" {
   ITEM
 }
 
+resource "aws_dynamodb_table_item" "test_client_base_permissions" {
+  table_name = aws_dynamodb_table.permissions_table.name
+  hash_key   = aws_dynamodb_table.permissions_table.hash_key
+  range_key  = aws_dynamodb_table.permissions_table.range_key
+
+  item = <<ITEM
+  {
+    "PK": {"S": "SUBJECT"},
+    "SK": {"S": "${aws_cognito_user_pool_client.e2e_test_client_base_permissions.id}"},
+    "Id": {"S": "${aws_cognito_user_pool_client.e2e_test_client_base_permissions.id}"},
+    "Type": {"S": "CLIENT"},
+    "Permissions": {"SS": ["READ_ALL_PUBLIC", "WRITE_ALL_PUBLIC"]}
```

Comment:
Why does it say base permissions and have READ_ALL_PUBLIC and WRITE_ALL_PUBLIC?

Reply:
Naming is a bit of a misnomer -> I think I want to have a low level permission role on which I can try various tests, and these were the lowest level permissions I could think of granting that don;t allow write/read to protected domain or admin access (data or user)